pkgs/top-level/aliases.nix

```diff
@@ -769,7 +769,9 @@ mapAliases ({
 
   ### K ###
 
-  k3d = throw "k3d has been removed because it was broken and has seen no release since 2016"; # Added 2022-01-04
+  # k3d was a 3d editing software k-3d - "k3d has been removed because it was broken and has seen no release since 2016" Added 2022-01-04
+  # now kube3d/k3d will take it's place
```

Comment:
This comment wouldn't have been necessary.

```suggestion
```

Reply:
It's just a bit of extra context because it's taking over an old alias
We can remove the comment when the alias is removed :shrug: